src/main/java/com/atharvadholakia/password_manager/utils/CaesarCipher.java

```diff
@@ -0,0 +1,28 @@
+package com.atharvadholakia.password_manager.utils;
+
+public class CaesarCipher {
+
+  public static String encrypt(String plaintext, int shift) {
+    StringBuilder encryptedText = new StringBuilder();
+
+    for (char c : plaintext.toCharArray()) {
+
+      if (Character.isUpperCase(c)) {
+        char encryptedChar = (char) ((c + shift - 'A') % 26 + 'A');
```

Comment:
The logic here is, we use a shift i.e how many times we shift the alphabet forward. For example a shift of 3 changes 'A' to 'D'. 
The modulo 26 is used to keep the shifting in the range of A-Z alphabets. This is useful when we want to shift lets say Z with a shift of 3. So it will wrap around to the beginning of the alphabets and not go out of the range. 'Z' --> 'C'